.github/workflows/test.yml

```
@@ -0,0 +1,213 @@
+        cache: 'pnpm'
+        
+    - name: Install dependencies
+      run: pnpm install --frozen-lockfile
+      
+    - name: Run adapter-electron tests
+      run: |
+        cd packages/adapter-electron
+        pnpm test
+        
+    - name: Run adapter-appwrite tests (if exists)
+              run: |
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 42: mapping values are not allowed in this context

(syntax-check)